index.html

```diff
@@ -289,7 +289,7 @@ <h2 class="mt-0 heading-border-top font-weight-normal">Get In Touch</h2>
 				<div class="row align-items-center">
 					<div class="col-md-12">
 						<h2 class="heading mb-3">Interested in Volunteering?</h2>
-						<p class="sub-heading mb-5 pb_color-light-opacity-8">Are you a SEBS or PCHS alumni interested in helping? Submit your volunteering application here!</p>
+						<p class="sub-heading mb-5 pb_color-light-opacity-8">Are you a SEBS or PCHS alumni interested in making a difference in other SEBS students' lives? Fill out the volunteering interest survey here!</p>
```

Comment:
I don't like how this reads. I prefer this updated language, but perhaps it would read better as: "Are you a SEBS or PCHS alumni interested in making a difference in the lives of upcoming SEBS graduates? Fill out the volunteering interest form here!"

Reply:
Fixed in 51967d8.